docs/examples/dialog_box.md

```diff
@@ -68,5 +68,34 @@ def test_num_input():
     )
     response = hardpy.run_dialog_box(dbx)
     assert response == 123.0
+
+
+@pytest.mark.case_name("Radiobutton")
+def test_radiobutton():
+    dbx = DialogBox(
+        dialog_text=f"Select one item (1) from several and click the Confirm button",
+        title_bar="Radiobutton example",
+        widget=DialogBoxWidget(
+            DialogBoxWidgetType.RADIOBUTTON,
+            {"options": ["one", "two", "three"]},
+        ),
+    )
+    response = run_dialog_box(dbx)
+    set_message(f"Selected item {response}")
+    assert response == "one", "The answer is not correct"
+
+
+@pytest.mark.case_name("Checkbox")
+def test_checkbox():
+    dbx = DialogBox(
+        dialog_text=f"Select one or more items (1) and click the Confirm button",
+        title_bar="Checkbox example",
+        widget=DialogBoxWidget(
+            DialogBoxWidgetType.CHECKBOX, {"options": ["one", "two", "three"]}
+        ),
+    )
+    response = run_dialog_box(dbx)
+    set_message(f"Selected item {response}")
+    assert response == '["one"]', "The answer is not correct"
```

Comment:
Need to show how to use a checkbox with multiple answers.

Reply:
Ready